app/views/single_page_subscriptions/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<% content_for :title, t("single_page_subscriptions.title") %>
+
+<%= render "govuk_publishing_components/components/heading", {
+  text: t("single_page_subscriptions.title"),
+  heading_level: 1,
+  font_size: "l",
+  margin_bottom: 6
+} %>
+
+<p class="govuk-body">
+  <%= t("single_page_subscriptions.account_required_description") %>
+</p>
+
+<%= render "govuk_publishing_components/components/heading", {
+  text: t("single_page_subscriptions.subheading"),
+  heading_level: 2,
+  font_size: "m",
+  margin_bottom: 6
+} %>
+
+<%= t("single_page_subscriptions.accounts_are_new_description_html") %>
+
+<%= render "govuk_publishing_components/components/inset_text", {
+} do
+  t("single_page_subscriptions.inset_description")
+end %>
+
+<%= t("single_page_subscriptions.usage_description_html") %>
+
+<%= form_tag single_page_new_session_path method: :post do %>
```

Comment:
You want the `explicit-cross-domain-links` on this form, to supply the parameters for the `redirect_with_analytics`